test/unit/providers/timer.provider.unit.ts

```diff
@@ -5,19 +5,22 @@
 
 import {expect} from '@loopback/testlab';
 import {TimerProvider} from '../../..';
+import {TimerFn, HighResTime} from '../../..';
 
 describe('TimerProvider (unit)', () => {
   it('returns current time given no start time', () => {
-    const timer = new TimerProvider().value();
-    const time = <[number, number]>timer();
+    const timer: TimerFn = new TimerProvider().value();
+    const time: HighResTime = <[number, number]>timer();
```

Comment:
Is `<[number, number]>` cast still needed? The next test below does not cast the result of `timer()`.